tools/chplvis/ConcurrencyView.cxx

```diff
@@ -0,0 +1,425 @@
+/*
+ * Copyright 2016 Cray Inc.
+ * Other additional copyright holders may be indicated within.
+ *
+ * The entirety of this work is licensed under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ *
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdio.h>
+#include <assert.h>
+
+#include "ConcurrencyView.h"
+#include "chplvis.h"
+
+#include "SubView.h"
+#include "SelectBrowser.h"
+
+#include <FL/fl_draw.H>
+#include <FL/fl_ask.H>
+
+// Callback for this view
+
+void backCallback (Fl_Widget *w, void *p) {
+  ConcurrencyView *view = (ConcurrencyView *)w->parent();
+  view->showTaskBox();
+  //printf ("backButtonClicked\n");
```

Comment:
I assume you're leaving these comments in for future debugging purposes. I would recommend either removing them or wrapping them with `if chplvis-debug { .. }` blocks (and adding some `chplvis-debug` that can be activated by Makefile flag or something)